app/services/users.js

```diff
@@ -0,0 +1,37 @@
+const { users } = require('../models'),
+  logger = require('../logger'),
+  errors = require('../errors'),
+  bcrypt = require('bcryptjs');
+const saltRounds = 10;
+
+exports.createUser = async data => {
+  try {
+    let user = await users.findOne({
+      where: {
+        email: data.email
+      }
+    });
+    let message = '';
+    if (user) {
+      message = `The user "${user.email}" already exists`;
+      logger.error(message);
+      return errors.userAlreadyExistsError(message);
```

Comment:
return is not necessary, work with `throw`

Reply:
I fix this in the next PR